great_expectations/_docs_decorators.py

```diff
@@ -134,9 +134,11 @@ def my_method(some_argument):
 
     This tag is added at import time.
     """
-    public_api_introspector.add(func)
-    existing_docstring = func.__doc__ if func.__doc__ else ""
```

Comment:
Minor change in ordering - let's fail fast if we don't have the docstring before adding to the introspector. Marginal performance gains doesn't really matter tbh